api/assessment.go

```diff
@@ -52,8 +52,15 @@ func (h AssessmentHandler) Get(ctx *gin.Context) {
 		_ = ctx.Error(result.Error)
 		return
 	}
+
```

Comment:
The preload on line 49 should have the Questionnaire relationships on the assessment populated, and you should be able to grab the required property from there rather than setting up a resolver.